browserTests/browserTest.js

```diff
@@ -61,13 +61,13 @@ test('Contrast does change', async (t) => {
   ;
 });
 
-fixture`Map tests`
+/* fixture`Map tests`
```

Comment:
Test not working so commented out?

Reply:
That tests if bus stops are visible on the map and we don't display those yet.

Reply:
Cool